includes/integration.php

```diff
@@ -577,8 +578,8 @@ public function prepare_product_hash($product, $variation_id = false, $variation
         }
         // fetch image URL
         $image_id = get_post_thumbnail_id($product_id);
-        $image = get_post($image_id);
-        if($image && $image->guid) $product_hash['image_url'] = $image->guid;
+        $image_url = wp_get_attachment_image_src($image_id, 'full')[0];
+        if($image_url) $product_hash['image_url'] = $image_url;
```

Comment:
Instead of copying and pasting the code around why not make a shared function?

Reply:
New function for img url is added.